consul/files/services.json

```diff
@@ -1,4 +1,3 @@
-{% from "consul/map.jinja" import consul with context %}
```

Comment:
This is good stuff. I've found that such statement will not work with Salt SSH and thin provisioning. Better to get rid of this.